packages/react-components/src/components/AvatarPersona.tsx

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { IPersonaProps, Persona, PersonaInitialsColor } from '@fluentui/react';
+import React, { useEffect } from 'react';
+
+/**
+ * Custom data attributes for the AvatarPersona component.
+ */
+export type AvatarPersonaData = {
+  /**
+   * Primary text to display, usually the name of the person.
+   */
+  text?: string;
+  /**
+   * Image URL to use, should be a square aspect ratio and big enough to fit in the image area.
+   */
+  imageUrl?: string;
+  /**
+   * The user's initials to display in the image area when there is no image.
+   * @defaultvalue Derived from `text`
+   */
+  imageInitials?: string;
+  /**
```

Comment:
The last two fields here seem to mix styling with content.

In component props, we should uniformly support the `styles` field for that instead.

This isn't a component prop though, which muddies the water a little bit.

If the intent is to provide a way to set foreground and background color *only* when displaying initials, one way would be to introduce a `PersonaInitials` component, and optionally expose `PersonaInitialsProps` here (which would inlude content, `text`, as well as the usual `styles`.)

Reply:
The current props exposed here have been borrowed from the actual `Persona` components. 
This allows us to achieve uniformity and makes it easier for people already familiar with Fluent UI to understand this component.

Exposing a `styles` object would defeat the purpose as it would allow CONTOSO to mess up with the styling of the component which is exactly what we are trying to avoid.